mkdocs.yml

```diff
@@ -73,7 +73,6 @@ extra_css:
 extra_javascript:
   - 'https://unpkg.com/lunr/lunr.js'
 markdown_extensions:
-  - codehilite
   - admonition
   - pymdownx.details
   - pymdownx.highlight
```

Comment:
```suggestion
  - pymdownx.highlight:
       pygments_lang_class: true
```

Reply:
error was because codehilite is deprecated, so the page was broken

Reply:
it is nice to have

Reply:
updated 👍